domain/robotouille.json

```diff
@@ -1004,6 +1008,75 @@
             "precons": [],
             "immediate_fx": [],
             "sfx": []
+        },
+        {
+            "name": "pan-fry",
```

Comment:
Similar with this action, remove it for this PR, especially since we changed the pan and pan-fry action in the fried eggs PR